src/Linux/Avalonia.LinuxFramebuffer/Input/LibInput/LibInputBackend.cs

```diff
@@ -29,15 +28,21 @@ public LibInputBackend()
             new Thread(()=>InputThread(ctx)).Start();
         }
 
-        
-        
         private unsafe void InputThread(IntPtr ctx)
         {
             var fd = libinput_get_fd(ctx);
             
             var timeval = stackalloc IntPtr[2];
 
-
+            if (!Directory.Exists("/dev/input"))
+            {
+                if (Logging.Logger.IsEnabled(Logging.LogEventLevel.Warning,LibInput))
+                {
+                    Logging.Logger.TryGet(Logging.LogEventLevel.Warning, LibInput)
+                        ?.Log(this, "Not connect any input device.");
+                }
```

Comment:
Minor: you don't need to double check IsEnabled, because TryGet will return null if logger is disabled.

Also, not sure if we should create logging areas per subfeatures. I.e. probably better to keep "X11Platform" without "/LibInput".